bylaws.tex

```diff
@@ -164,32 +164,32 @@
 \bsubsection{The Introductory Packet}
 Each Introductory Process participant, after the first week, is given two (2) weeks to return a document that should contain the following:
 \begin{itemize}
-	\item A signature from all Active/Introductory Resident members, each Executive Board member, and fifteen (15) of any combination of the following:
+	\item A signature from all Active/Introductory Resident members, each Executive Board member, and fifteen (15) of any combination of the following:.
```

Comment:
There are a few issues like this throughout the document. Might want to modify the script a bit more.